migtests/scripts/run-test.sh

```diff
@@ -67,15 +67,14 @@ main() {
 	step "Export data."
 	# false if exit code of export_data is non-zero
 	export_data || { 
-		# Print debezium.log file if present and exportDataDone flag file is not present
-		if [ -f "${EXPORT_DIR}/logs/debezium.log" ] && [ ! -f "${EXPORT_DIR}/metainfo/flags/exportDataDone" ]
+		if [ ! -f "${EXPORT_DIR}/metainfo/flags/exportDataDone" ]
```

Comment:
I think we can get rid of this check as well. I guess, earlier this was added to detect export data failure. With the way we arranged `export_data() || {`, when the control flow reaches line 70, we "know" that export data filed).

Reply:
Yeah I was thinking the same